Index: src/Server/QueryController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Server;\r\n\r\npublic class QueryController {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Server/QueryController.java b/src/Server/QueryController.java
--- a/src/Server/QueryController.java	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
+++ b/src/Server/QueryController.java	(date 1672164656640)
@@ -1,4 +1,24 @@
 package Server;
 
+
+import Protocols.Packet;
+import Server.Tours.ToursController;
+
 public class QueryController {
-}
+    static Packet query_request(Packet packet){
+        switch (Packet.CODES.get(packet.getCode()).toLowerCase()){
+            case "tours"->{
+                Packet packet_return = ToursController.INSTANCE.process(packet);
+                packet_return.setCode(packet.getCode());
+                return packet;
+            }
+            case "trips"->{
+                return null;
+            }
+            case "ticket"->{
+                return null;
+            }
+        }
+        return null;
+    }
+}
\ No newline at end of file
Index: src/MainClient.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.io.IOException;\r\nimport java.net.Inet4Address;\r\n\r\npublic class MainClient {\r\n    public static void main(String[] args) {\r\n        Inet4Address ip;\r\n        int port;\r\n\r\n        try{\r\n\r\n        } catch (IOException exception){\r\n\r\n        }\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/MainClient.java b/src/MainClient.java
--- a/src/MainClient.java	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
+++ b/src/MainClient.java	(date 1672164576945)
@@ -5,11 +5,5 @@
     public static void main(String[] args) {
         Inet4Address ip;
         int port;
-
-        try{
-
-        } catch (IOException exception){
-
-        }
     }
 }
Index: target/classes/Заметки.txt
===================================================================
diff --git a/target/classes/Заметки.txt b/target/classes/Заметки.txt
deleted file mode 100644
--- a/target/classes/Заметки.txt	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
+++ /dev/null	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
@@ -1,51 +0,0 @@
-import java.lang.reflect.InvocationTargetException;
-import java.sql.*;
-import MVC.JMainFrame;
-import Server.Tours.ToursView;
-
-public class Server.DBWorker {
-
-        private static final String URL = "jdbc:mysql://localhost:3306/kursa";
-        private static final String USERNAME = "root";
-        private static final String PASSWORD = "9228lalala";
-        public static void Server.DBWorker()  {
-            //JMainFrame jMainFrame = new JMainFrame();
-            //jMainFrame.set_panel(ToursView.INSTANCE);
-
-
-
-            try {
-                Driver driver = (Driver) Class.forName("com.mysql.cj.jdbc.Driver").getDeclaredConstructor().newInstance();
-                DriverManager.registerDriver(driver);
-
-
-//                if (!connection.isClosed()){
-//                    System.out.println("Соединение с БД установлено!");
-//                }
-//                connection.close();
-//                if (connection.isClosed()){
-//                    System.out.println("Соединение с БД закрыто!");
-//                }
-            } catch (SQLException | ClassNotFoundException | NoSuchMethodException | InstantiationException |
-                     IllegalAccessException | InvocationTargetException e){
-                System.out.println("ЖОПА");
-            }
-
-            try (Connection connection = DriverManager.getConnection(URL,USERNAME,PASSWORD); Statement statement = connection.createStatement()) {
-                //System.out.println(connection.isClosed());
-                //statement.execute("insert into tours (id,city) values(9,'Янечорт');");
-                //int change =  statement.executeUpdate("UPDATE tours SET city='Я чорт' where id > 5"); // Кол-во изменений
-                //System.out.println(change);
-                //ResultSet set = statement.executeQuery("SELECT * from tickets");
-                //System.out.println(set);
-                //statement.addBatch(""); // Добавление набора команд (некая транзакция) в batch
-                //statement.executeBatch(); // Выполнения набора команд
-                //statement.clearBatch(); // Очистка набора команд
-                if (!connection.isClosed()) System.out.println("Соединение установлено!");
-            } catch (SQLException e) {
-                throw new RuntimeException(e);
-            }
-
-
-        }
-}
Index: target/classes/resources/test.sql
===================================================================
diff --git a/target/classes/resources/test.sql b/target/classes/resources/test.sql
deleted file mode 100644
--- a/target/classes/resources/test.sql	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
+++ /dev/null	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
@@ -1,1 +0,0 @@
-insert into tours (id,city) values(8,'Ячорт');
\ No newline at end of file
Index: target/classes/Server/serverApp.fxml
===================================================================
diff --git a/target/classes/Server/serverApp.fxml b/target/classes/Server/serverApp.fxml
deleted file mode 100644
--- a/target/classes/Server/serverApp.fxml	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
+++ /dev/null	(revision 4b18f403a0813b827cdde08cc2affeab9d973864)
@@ -1,81 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.SplitPane?>
-<?import javafx.scene.control.TableColumn?>
-<?import javafx.scene.control.TableView?>
-<?import javafx.scene.control.TextField?>
-<?import javafx.scene.layout.AnchorPane?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.VBox?>
-
-<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1" fx:controller="Server.AppController">
-   <children>
-      <SplitPane dividerPositions="0.29797979797979796" prefHeight="400.0" prefWidth="600.0">
-        <items>
-          <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="160.0" prefWidth="100.0">
-               <children>
-                  <VBox maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minHeight="-Infinity" minWidth="-Infinity" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
-                     <children>
-                        <VBox alignment="CENTER" minHeight="-Infinity" minWidth="-Infinity" prefHeight="363.0" prefWidth="175.0" spacing="10.0">
-                           <children>
-                              <TextField fx:id="numField" disable="true" />
-                              <TextField fx:id="textField" disable="true" />
-                              <Button fx:id="btn_add" disable="true" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" mnemonicParsing="false" style="-fx-background-color: #559955;" text="Добавить +" />
-                              <Button fx:id="btn_add1" disable="true" layoutX="20.0" layoutY="268.0" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" mnemonicParsing="false" style="-fx-background-color: #559955;" text="Изменить" />
-                              <Button fx:id="btn_delete" disable="true" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" mnemonicParsing="false" prefHeight="25.0" prefWidth="190.0" style="-fx-background-color: #995555;" text="Удалить -" />
-                           </children>
-                           <padding>
-                              <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
-                           </padding>
-                        </VBox>
-                        <HBox alignment="CENTER" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minHeight="-Infinity" minWidth="-Infinity" prefHeight="31.0" prefWidth="175.0" spacing="5.0">
-                           <children>
-                              <Button fx:id="btn_back" alignment="CENTER" disable="true" graphicTextGap="0.0" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" minHeight="-Infinity" minWidth="-Infinity" mnemonicParsing="false" style="-fx-background-color: #ffaaaa;" text="&lt;- Back" />
-                              <Button fx:id="btn_enter" alignment="CENTER" disable="true" graphicTextGap="0.0" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" mnemonicParsing="false" style="-fx-background-color: #aaffaa;" text="Перейти -&gt;" />
-                           </children>
-                        </HBox>
-                     </children>
-                  </VBox>
-               </children>
-            </AnchorPane>
-          <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="160.0" prefWidth="100.0">
-               <children>
-                  <TableView fx:id="toursTable" onMouseClicked="#tableMouseClickHandler" prefHeight="398.0" prefWidth="419.0">
-                    <columns>
-                      <TableColumn fx:id="toursId" prefWidth="75.0" text="toursId" />
-                      <TableColumn fx:id="toursCity" prefWidth="421.0" text="toursCity" />
-                        <TableColumn fx:id="toursPrice" prefWidth="75.0" text="toursPrice" />
-                    </columns>
-                     <columnResizePolicy>
-                        <TableView fx:constant="CONSTRAINED_RESIZE_POLICY" />
-                     </columnResizePolicy>
-                  </TableView>
-                  <TableView fx:id="tripsTable" onMouseClicked="#tableMouseClickHandler" prefHeight="398.0" prefWidth="419.0" visible="false">
-                     <columns>
-                        <TableColumn fx:id="tripsId" prefWidth="75.0" text="tripsId" />
-                        <TableColumn fx:id="tripsTour" prefWidth="421.0" text="tripsTour" />
-                        <TableColumn fx:id="tripsPrice" prefWidth="75.0" text="tripsPrice" />
-                        <TableColumn fx:id="tripsDatetime" prefWidth="75.0" text="tripsDatetime" />
-                     </columns>
-                     <columnResizePolicy>
-                        <TableView fx:constant="CONSTRAINED_RESIZE_POLICY" />
-                     </columnResizePolicy>
-                  </TableView>
-                  <TableView fx:id="ticketsTable" onMouseClicked="#tableMouseClickHandler" prefHeight="398.0" prefWidth="419.0" visible="false">
-                     <columns>
-                        <TableColumn fx:id="ticketId" prefWidth="75.0" text="ticketId" />
-                        <TableColumn fx:id="ticketTrip" prefWidth="421.0" text="ticketTrip" />
-                        <TableColumn fx:id="ticketName" prefWidth="75.0" text="ticketName" />
-                     </columns>
-                     <columnResizePolicy>
-                        <TableView fx:constant="CONSTRAINED_RESIZE_POLICY" />
-                     </columnResizePolicy>
-                  </TableView>
-               </children>
-            </AnchorPane>
-        </items>
-      </SplitPane>
-   </children>
-</AnchorPane>
