Index: src/MVC/JMainFrame.java
===================================================================
diff --git a/src/MVC/JMainFrame.java b/src/MVC/JMainFrame.java
deleted file mode 100644
--- a/src/MVC/JMainFrame.java	
+++ /dev/null	
@@ -1,23 +0,0 @@
-package MVC;
-
-import javax.swing.*;
-
-public class JMainFrame extends JFrame {
-
-    JPanel currentPanel = null;
-    public JMainFrame(){
-        // создание окна
-        super("SERVERSIDE");
-        // выход при закрытии окна
-        setDefaultCloseOperation(EXIT_ON_CLOSE);
-        setSize(600, 800);
-        setVisible(true);
-    }
-
-    public void set_panel(JPanel panel){
-        if (this.currentPanel != null)
-            this.remove(this.currentPanel);
-        this.currentPanel = panel;
-        setContentPane(panel);
-    }
-}
Index: .idea/misc.xml
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
deleted file mode 100644
--- a/.idea/misc.xml	
+++ /dev/null	
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="19" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
-</project>
\ No newline at end of file
